lib/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@storyblok/nuxt",
+  "version": "2.0.0",
+  "description": "Storyblok Nuxt.js module",
+  "license": "MIT",
+  "contributors": [
+    {
+      "name": "Alexander Feiglstorfer <delooks@gmail.com>"
+    }
+  ],
+  "main": "./module.js",
+  "repository": "https://github.com/storyblok/storyblok-nuxt",
+  "scripts": {
+    "test": "npm run test:unit && npm run test:e2e",
+    "test:unit": "jest __tests__",
```

Comment:
These commands are failing because we don't have jest installed here neither unit tests. So, I don't see a problem removing the logic regarding unit tests. What do you think?

Reply:
True, for this integration tests makes more sense, so that's why we're not using unit. We can remove it